samples/msal-angular-samples/angular-b2c-sample/src/app/app.component.ts

```diff
@@ -28,9 +29,21 @@ export class AppComponent implements OnInit, OnDestroy {
         private msalBroadcastService: MsalBroadcastService
     ) { }
 
-    ngOnInit(): void {
+    async ngOnInit(): Promise<void> {
         this.isIframe = window !== window.parent && !window.opener; // Remove this line to use Angular Universal
 
+        await (this.authService.instance as PublicClientApplication).initialize();
```

Comment:
What is the reason for initializing PCA and using handleRedirectPromise instead of subscribing to the handlerRedirectObservable?